shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/data/local/model/SurveyRealmObject.kt

```diff
@@ -34,8 +34,10 @@ open class SurveyRealmObject : RealmObject {
 }
 
 fun SurveyRealmObject.toSurvey() = Survey(
+    id,
     title,
     description,
     isActive,
-    coverImageUrl
+    coverImageUrl,
+    emptyList()
```

Comment:
When mapping the object/model, I would recommend adding properties name, it would be clear when you pass empty/null fields such as: "", emtptyList(), null, .etc

```
id = id,
title = title,
...
questions = emptyList()
```

Reply:
Fixed all files in https://github.com/suho/kmm-ic/pull/110/commits/3dd86b6dd066888cac19cd03ccc631346926311b